src/server/api/schema.js

```diff
@@ -1979,6 +1979,50 @@ const rootMutations = {
       return result;
     },
 
+    updateCampaignAutosendingLimit: async (
+      _ignore,
+      { campaignId, limit },
+      { user }
+    ) => {
+      const id = parseInt(campaignId, 10);
+
+      const campaign = await r
+        .knex("all_campaign")
+        .where({ id })
+        .first(["organization_id"]);
+
+      const organizationId = campaign.organization_id;
+      await accessRequired(user, organizationId, "ADMIN", true);
+
+      const {
+        rows: [updatedCampaign]
+      } = await r.knex.raw(
+        `
+          update all_campaign
+          set
+            autosend_limit = ?::int,
+            autosend_limit_max_contact_id = (case
+              when ?::int is not null then (
+                select max(id)
+                from (
+                  select id
+                  from campaign_contact
+                  where campaign_id = ?::int
+                  order by id asc
+                  limit ?::int
+                ) campaign_contact_ids
+              )
+              else null
+            end)
```

Comment:
Is the reason we handle the case where limit is null so that this query can double for setting autosend_limit/max_contact_id to null?

nit: I think it'd be easier to read as 2 separate queries